### ISSUE TITLE
Reverse `embark-quit-after-action` for `C-. RET` only

### ISSUE THREAD

**Issue by astoff** [oantolin/embark#611]
I keep the default `embark-quit-after-action=t`, and I think the only occasion I really have to use the prefix arg of `embark-act` is when I'm going to call the default action. And, conversely, I usually have no reason to do `C-. RET` without a prefix arg.

So is there a way to make that happen by default?

**Comment by minad** [comment]
Why not use `M-.`? You could write your own command which let-binds `embark-quit-after-action`.

**Comment by oantolin** [comment]
I agree with @minad that you can give yourself an even better keybinding by defining your own variant of embark-dwim.

But maybe it's worth adding a `:default-action` key to `embark-quit-after-action`°

**Comment by oantolin** [comment]
> On the other hand the option you suggested might be a bit too much of a special case. Note also that I would want the opposite, :`except-for-default-action`.

I'm not sure but I suspect you misunderstood slightly what I was proposing. I wasn't suggesting we allow `:default-action` as a possible _value_ of `embark-quit-after-action`, but rather as a possible _key_. The variable `embark-quit-after-action` can be set to a boolean or to an alist mapping actions to a boolean, to specify on a per-action basis whether you want to quit or not.

The alist case can have the special key `t` to specify the default quitting behavior. For example, `(setq embark-quit-after-action '((kill-buffer . t) (t . nil)))` will make `embark-act` quit when the action is `kill-buffer` but default to not quitting.

There is currently no easy way to specify what you want for the default action, since the default action is not a single action. That's why I suggested adding a second special key, `:default-action`, that you could map to `nil`.

**Comment by oantolin** [comment]
But probably @minad's suggestion is even better for you since you get a shorter keybinding than `C-. RET`.

**Comment by astoff** [comment]
Ah okay, indeed I misunderstood you. I think in this case the `:default-action` key is an addition that makes sense (regardless of Daniel's suggestion which makes sense as well).
